packages/commonwealth/client/scripts/views/pages/CommunityGroupsAndMembers/Members/MembersSection/MembersSection.tsx

```diff
@@ -105,6 +107,42 @@ const MembersSection = ({
   };
   const filteredMember = removeDuplicateAddresses(filteredMembers);
 
+  // Add function to determine icon based on community base and wallet type
+  const getChainIcon = (address: AddressInfo): CustomIconName => {
```

Comment:
this function is defined 4 times in the code, please move it to the util, helper and reuse it everywhere else